Remove previewDevice from #Preview to clear warning

## Summary
`xcodebuild test` emits a warning that `previewDevice` is ignored when used inside a `#Preview` macro.

## Evidence
```
/Users/sridhar/conductor/workspaces/NammaMeter/lahore/NammaMeter/SettingsView.swift:96:6: warning: previewDevice is ignored in a #Preview macro. Use the device picker at the bottom of the Canvas to change which device type is used for the preview. (from macro "Preview")
    .previewDevice("iPhone 17 Pro")
     ^
```

## Context
Observed while running:
```
xcodebuild test -project NammaMeter.xcodeproj -scheme NammaMeter -destination "platform=iOS Simulator,name=iPhone 12"
```

## Acceptance Criteria
- The warning no longer appears in build/test output.
- `#Preview` uses supported configuration (remove `.previewDevice(...)` or move device selection to Xcode’s preview picker).

--- a/NammaMeter/SettingsView.swift
+++ b/NammaMeter/SettingsView.swift
@@ -93,5 +93,4 @@ struct LabeledValue: View {
   SettingsView()
     .environment(SettingsStore())
     .environment(MeterStore())
-    .previewDevice("iPhone 17 Pro")
 }

--- a/NammaMeterTests/MeterStoreTests.swift
+++ b/NammaMeterTests/MeterStoreTests.swift
@@ -79,6 +79,14 @@ final class MeterStoreTests: XCTestCase {
 
     meterStore.startTrip(settings: settings)
 
+    var dayComponents = DateComponents()
+    dayComponents.year = 2026
+    dayComponents.month = 2
+    dayComponents.day = 3
+    dayComponents.hour = 10
+    let dayDate = Calendar.autoupdatingCurrent.date(from: dayComponents)!
+    meterStore.refreshTimeBasedConditions(reference: dayDate)
+
     XCTAssertTrue(meterStore.isOnTrip)
     XCTAssertEqual(meterStore.tripState, .inProgress)
     XCTAssertEqual(meterStore.fare, settings.minFare)
@@ -114,6 +122,13 @@ final class MeterStoreTests: XCTestCase {
     let settings = MeterSettings.bengaluruDefault
 
     meterStore.startTrip(settings: settings)
+    var dayComponents = DateComponents()
+    dayComponents.year = 2026
+    dayComponents.month = 2
+    dayComponents.day = 3
+    dayComponents.hour = 10
+    let dayDate = Calendar.autoupdatingCurrent.date(from: dayComponents)!
+    meterStore.refreshTimeBasedConditions(reference: dayDate)
     XCTAssertTrue(tripStore.trips.isEmpty)
 
     meterStore.stopTrip(tripStore: tripStore)
@@ -241,6 +256,13 @@ final class MeterStoreTests: XCTestCase {
     settings.baseFare = 36
 
     meterStore.startTrip(settings: settings)
+    var dayComponents = DateComponents()
+    dayComponents.year = 2026
+    dayComponents.month = 2
+    dayComponents.day = 3
+    dayComponents.hour = 10
+    let dayDate = Calendar.autoupdatingCurrent.date(from: dayComponents)!
+    meterStore.refreshTimeBasedConditions(reference: dayDate)
 
     // Even with baseFare < minFare, fare should be minFare
     XCTAssertEqual(meterStore.fare, settings.minFare)

